css/styles.css

```diff
@@ -65,32 +61,40 @@ a {
     color: black; 
 }
 
+i {
+    font-size: 1.5rem; 
+}
+
+.dot {
+    font-family: Georgia, 'Times New Roman', Times, serif;
+}
+
 
 
```

Comment:
maybe you could have a navbar or home button that stays? or even a back to top link to save the scroll? :)

Reply:
Maybe one day!!! I'll add that to the roadmap 👍